src/gtest/test_zip32.cpp

```diff
@@ -132,3 +132,41 @@ TEST(ZIP32, TestVectors) {
         m_1_2hv_3.DefaultAddress().d,
         testing::ElementsAreArray({ 0x03, 0x0f, 0xfb, 0x26, 0x3a, 0x93, 0x9e, 0x23, 0x0e, 0x96, 0xdd }));
 }
+
+TEST(ZIP32, ParseZip32KeypathAccount) {
+    std::string sAccount = "m/32'/1234'/5'";
+    EXPECT_TRUE(libzcash::ParseZip32KeypathAccount(sAccount).has_value());
+    EXPECT_EQ(libzcash::ParseZip32KeypathAccount(sAccount).value(), 5);
```

Comment:
Technically this style opens up the possibility of non-determinism going undetected, where the first call returns a nonsensical non-None value. I'd prefer:
```suggestion
    auto parsedAccount = libzcash::ParseZip32KeypathAccount(sAccount);
    EXPECT_TRUE(parsedAccount.has_value());
    EXPECT_EQ(parsedAccount.value(), 5);
```
and so on below. This could be refactored into a (lambda) function.